pkg/api/types.go

```diff
@@ -1110,6 +1110,26 @@ func (m *MasterProfile) HasAvailabilityZones() bool {
 	return m.AvailabilityZones != nil && len(m.AvailabilityZones) > 0
 }
 
+// IsUbuntu1604 returns true if the master profile distro is based on Ubuntu 16.04
+func (m *MasterProfile) IsUbuntu1604() bool {
```

Comment:
Masters only support Linux and don't have an `osType` field so we don't need to check the OS